### ISSUE TITLE
Missing compile script commands documentation

### ISSUE THREAD

**Issue by JohnTheCoolingFan** [armbian/documentation#431]
Page: https://docs.armbian.com/Developer-Guide_Build-Options/

Some commands are missing but can be found in `lib/functions/cli/commands.sh`. For example, `rewrite-uboot-patches` is documented (listed) but `rewrite-kernel-patches` is not.

Also, `dts-check` is listed but is currently missing ([reverted](https://github.com/armbian/build/commit/aa9fea779702834a2a477ee2ad361e30a81bc7a6))

**Comment by igorpecovnik** [comment]
> Some commands are missing

Yes, unfortunately. We are aware of. Any PR on the topic will be very helpful.

**Comment by igorpecovnik** [comment]
Both points are addressed on the current [commands reference](https://docs.armbian.com/build-framework/commands/advanced/).

- `rewrite-kernel-patches` has its own section there now, alongside `rewrite-uboot-patches`, and is also referenced from the patching switches and user-configuration pages.
- `dts-check` is documented with an example. The revert you linked has since been undone — it is back in `lib/functions/cli/commands.sh`, so listing it is correct again.

Closing as resolved. If you spot other commands in `commands.sh` that are still missing from the docs, a fresh issue listing them would be welcome.
